package.json

```diff
@@ -39,7 +39,10 @@
       "baseURL": "src",
       "lib": "src/classmentors"
     },
-    "dependencies": {},
+    "dependencies": {
+      "c3": "npm:c3@^0.4.11",
+      "d3": "npm:d3@^4.2.2"
```

Comment:
`jspm inspect npm:d3` might only show the one your module require (a bug I think).

`jspm inspect | grep d3` should show the two versions installed.